util/version/Cargo.toml

```diff
@@ -13,13 +13,13 @@ build = "build.rs"
 track = "nightly"
 
 # Indicates a critical release in this track (i.e. consensus issue)
-critical = false
+critical = true
```

Comment:
When do we switch this back to `false` afterwards?

Reply:
That's a bit unclear for me as well :)
I suppose we should flip it back when bumping new version, but clients updating from say 1.x.N to 1.x.(N+2) should still consider that critical, even though `N+2` is not. I don't think that logic is implemented in the updater currently, but don't consider that a blocker.